umatrix2ublock.py

```diff
@@ -62,7 +62,7 @@
 
         ub_action = translate_action[action]
 
-        ub_rule = f"{src} {dest} {ub_request} {ub_action}"
+        ub_rule = "{src} {dest} {ub_request} {ub_action}"
```

Comment:
@Tealk I would recommend using Python 3.6+ and keep using the code as it is. If for some reason you cannot migrate, you could change it to
```suggestion
        ub_rule = " ".join((src, dest, ub_request, ub_action))
```